.github/scripts/install_deps.sh

```diff
@@ -1,4 +1,11 @@
 apt-get install -y cmake g++
+
+# install protoc
 wget https://github.com/protocolbuffers/protobuf/releases/download/v21.10/protoc-21.10-linux-x86_64.zip
 unzip protoc-21.10-linux-x86_64.zip -d .local
-mv "$(pwd)/.local/bin/protoc" /bin/
\ No newline at end of file
+mv "$(pwd)/.local/bin/protoc" /bin/
+
+# install grcov
+wget https://github.com/mozilla/grcov/releases/download/v0.8.13/grcov-x86_64-unknown-linux-gnu.tar.bz2
+tar -xjvf grcov-x86_64-unknown-linux-gnu.tar.bz2
+mv grcov /bin/
```

Comment:
Add the new line whose line number is 12.

Reply:
There is a new line. Github hides it.

Reply:
OK, I see